packages/cli/src/ceramic-daemon.ts

```diff
@@ -799,6 +804,32 @@ export class CeramicDaemon {
     })
   }
 
+  async validateAdminRequest(req: Request, res: Response, next: NextFunction): Promise<void> {
```

Comment:
yeah a general one should look something like that, the pins requests ended all being the same and using headers (since no body, except some optional args), this is one point of inconsistency though